projects/ngx-intl-tel-input/src/lib/ngx-intl-tel-input.validator.ts

```diff
@@ -3,18 +3,25 @@ import * as lpn from 'google-libphonenumber';
 
 export const phoneNumberValidator = (control: FormControl) => {
 	const id = control.value && control.value.id ? control.value.id : 'phone';
-	const el = document.getElementById(id) ? document.getElementById(id) : undefined;
+	const el = document.getElementById(id) ? (<HTMLInputElement>document.getElementById(id)) : undefined;
```

Comment:
There's something wrong here. Is this a typo?

Reply:
No, I don't think so, I used TypeScript type assertion operator to make it infer from HTMLInputElement interface to access setCustomValidity method without errors.

Reply:
Interesting indeed will check ASAP.